### ISSUE TITLE
Wrong Exception names

### ISSUE THREAD

**Issue by defuse** [defuse/php-encryption#113]
```
                if ($read === false) {
                    throw new Ex\CannotPerformOperation(
                        'Could not read input file during decryption'
                    );
                }

```

That's the wrong name, it's called `CannotPerformOperationException`.

Unfortunately, testing will not discover these errors, so we have to carefully check manually.

**Comment by defuse** [comment]
The problem is usually the missing `Exception` on the end. Here are the commands to run to check:

```
grep -R CannotPerform . | grep -v OperationException
grep -R InvalidCiphertext . | grep -v CiphertextException
grep -R CryptoTestFailed . | grep -v FailedException
grep -R ExceptionException .
```

The last command finds search-and-replace errors introduced while fixing this problem before (there are some!).

**Comment by defuse** [comment]
Here's a command to check for uses of those exception classes without the namespace prefix:

```
grep -R 'CannotPerform\|InvalidCiphertext\|CryptoTestFailed' | grep -v '\\Defuse\\Crypto\\Exception' | grep -v 'Ex\\'
```

**Comment by defuse** [comment]
This is fixed in #116.
